tools/scripts/ansible/server-playbook.yml

```diff
@@ -7,54 +7,59 @@
     - { role: geerlingguy.mysql }
     - { role: geerlingguy.nginx }
   tasks:
+    - name: Deploy Tomcat
```

Comment:
This is just a placeholder
Really probably we should have separate playbooks for tomcat, mysql, nginx, cldr, unix/cldradmin
-- all included in setup-playbook.yml

Reply:
"tomcat-playbook.yml" would resemble part of "molecule/resources/converge.yml", see:
https://github.com/robertdebock/ansible-role-tomcat

Reply:
cf. https://robertdebock.nl/2019/02/21/ways-to-include-roles-in-roles.html